test/library/packages/Python/examples/numba/apply.precomp

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+FILE_DIR=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+$FILE_DIR/../../skipIfAndInstallPackage.sh $FILE_DIR numba
```

Comment:
I'm wondering if `skipIfAndInstallPackage.sh` needs to be renamed and/or adjusted to not return true/false for its use here, since precomp doesn't need it to do that and I think you're adjusting every use of it.

Reply:
I don't think the contents need to change, in my opinion it can still return True/False based on if the package is present or not. That will help with debugging.

But renaming it for better clarify, I will do that. Perhaps `checkAndInstallPackage.sh`?

Reply:
Works for me!